modeling-cmds/src/def_enum.rs

```diff
@@ -633,6 +633,13 @@ define_modeling_cmd_enum! {
             pub adjust_camera: bool,
         }
 
+        /// Set the clear color of the scene.
```

Comment:
Could you please explain what "the clear color" means?

Reply:
I have changed this to SetBackgroundColor as a more descriptive name